### PR TITLE
Fix Typos in ref-based RNAseq tutorial

### PR DESCRIPTION
**Pull request by lldelisle**
Fix typos and add tags

### CODE REVIEW

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 1]
```diff
@@ -1387,7 +1390,7 @@ DESeq2 requires to provide for each factor, counts of samples in each category.
 >            - *"Find pattern"*: `(.*)_(.*)_(.*)`
 >            - *"Replace with"*: `\1_\2_\3\tgroup:\2\tgroup:\3`
 >
->     This step adds an additional columns that can be used with the ``Tag elements from file`` tool
+>     This step adds an additional columns that can be used with the **Tag elements from file** {% icon tool %} tool
```
Comment (hexylena) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md:1393]:
```suggestion
>     This step adds an additional columns that can be used with the {% tool [Tag elements from file](__TAG_FROM_FILE__) %}  tool
```

`{% tool Tag Elements from file %}` should work as equivalent syntax and makes sure you get it right, but we can make it better by linking the tool maybe. https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#tool-links

**Review by hexylena — APPROVED** [review]

File: topics/transcriptomics/tutorials/ref-based/tutorial.md (hunk 1 above)
Comment (lldelisle) [reply topics/transcriptomics/tutorials/ref-based/tutorial.md]: Oups, I forgot there was a commit suggestion button...

**Comment by hexylena** [comment]
the other power you get @lldelisle is to merge your own PRs (if someone's approved them already.) Once the tests finish please feel free to merge it!

**Comment by lldelisle** [comment]
Thanks, I already knew this power but I feel more confident when the reviewer gives me his/her approval...

**Pull request merged**